Backends/common/ports.lisp

```diff
@@ -193,3 +193,16 @@
               (or (try-translators tr1 nil)
                   (try-translators (find-presentation-translators selected-type requested-type table-2)
                                    tr1)))))))))
+
+;;; Port macros
+
+(defmacro with-port ((port-var server &rest args &key &allow-other-keys)
+                     &body body)
+  `(invoke-with-port (lambda (,port-var) ,@body) ,server ,@args))
```

Comment:
I see that this is used in backends via `climi::with-port`, in this case we should export the symbol from climb package to avoid such access.

Reply:
Done.